### ISSUE TITLE
Thanks for the effort, don't understand why ebay is doing this now

### ISSUE THREAD

**Issue by Mike-the-one** [eBay/trading-api-java-sdk#6]
This lib does NOT work on java 17+ because of some of the dependencies does not work for higher version java.
Ant 1.6 is way too old, even 1.9 reaches EOL
Same as Java 1.8, EOL 2 years ago.

Why spend time on something that does not work for current java?
I would give you thumbs up if you can remove old dependencies and make the whole thing work for java 17+

**Comment by Craftoncu** [comment]
+1

**Comment by lrydl** [comment]
I had the same problem, and his developer website was really bad.

**Comment by mouyang** [comment]
@Mike-the-one I am in a similar situation as you as I was hoping this would "just work" on Android.  It didn't.  I played around with making the necessary changes to get it work on Java 11 and I encountered a lot of issues.

**Comment by mouyang** [comment]
@Mike-the-one @Craftoncu @lrydl I've been playing around rearranging things to build with Maven on Java 11.  Would you be interested in helping to test it out?  Will provide repo link, if desired.

**Comment by Mike-the-one** [comment]
Sure! do you have the link to your repo?

**Comment by mouyang** [comment]
Repo Link is below.  If you have any questions about the repo, let me know.  It's very raw and I tried to do the minimum needed to get a successful build, but in short it is meant to work in parallel with a clone of this repo.  The hope is it will eventually replace what is here currently.

on Github mouyang/ebay-trading-api-java-sdk ...  https://github.com/mouyang/ebay-trading-api-java-sdk

**Comment by Mike-the-one** [comment]
404

**Comment by mouyang** [comment]
> 404

Sorry about that. I fixed the link above.

**Comment by mouyang** [comment]
@Mike-the-one the repo link has been fixed.  Did you get a chance to look at it?  If you're still having issues, it should be visible in my Github profile.

**Comment by Mike-the-one** [comment]
Sorry was too busy, will give it a try this week or earlier next week.

**Comment by Mike-the-one** [comment]
So I looked at it, does your port create jar files that I can import to my project to replace ebay's jars?

**Comment by mouyang** [comment]
Yes.  It's still a WIP.  The intent is to restructure the repo into a Maven/Gradle project and build JARs from source by API version (and eventually Java LTS version).

    On Tuesday, November 11, 2025, 12:47:50 PM EST, Mike-the-one ***@***.***> wrote:  
 
 Mike-the-one left a comment (eBay/trading-api-java-sdk#6)
So I looked at it, does your port create jar files that I can import to my project to replace ebay's jars?

**Comment by mouyang** [comment]
@Mike-the-one @Craftoncu @lrydl it's been a while and I admittedly haven't had a chance to look at this myself.  Do you have feedback on my repo?  I may have some time to look into this, even if for no other reason than to experiment with AI coding tools.
